autowebcompat/network.py

```diff
@@ -45,42 +46,9 @@ def create_mlp(input_shape):
 
 
 def create_vgg16_network(input_shape):
-    input = Input(shape=input_shape)
-
-    # Block 1
-    x = Conv2D(64, (3, 3), padding='same', activation='relu')(input)
-    x = Conv2D(64, (3, 3), padding='same', activation='relu')(x)
-    x = MaxPooling2D(pool_size=(2, 2), strides=(2, 2))(x)
-
-    # Block 2
-    x = Conv2D(128, (3, 3), activation='relu', padding='same')(x)
-    x = Conv2D(128, (3, 3), activation='relu', padding='same',)(x)
-    x = MaxPooling2D(pool_size=(2, 2), strides=(2, 2))(x)
-
-    # Block 3
-    x = Conv2D(256, (3, 3), activation='relu', padding='same')(x)
-    x = Conv2D(256, (3, 3), activation='relu', padding='same')(x)
-    x = Conv2D(256, (3, 3), activation='relu', padding='same')(x)
-    x = MaxPooling2D(pool_size=(2, 2), strides=(2, 2))(x)
-
-    # Block 4
-    x = Conv2D(512, (3, 3), activation='relu', padding='same')(x)
-    x = Conv2D(512, (3, 3), activation='relu', padding='same')(x)
-    x = Conv2D(512, (3, 3), activation='relu', padding='same')(x)
-    x = MaxPooling2D(pool_size=(2, 2), strides=(2, 2))(x)
-
-    # Block 5
-    x = Conv2D(512, (3, 3), activation='relu', padding='same')(x)
-    x = Conv2D(512, (3, 3), activation='relu', padding='same')(x)
-    x = Conv2D(512, (3, 3), activation='relu', padding='same')(x)
-    x = MaxPooling2D(pool_size=(2, 2), strides=(2, 2))(x)
-
-    x = Flatten()(x)
-    x = Dense(4096, activation='relu')(x)
-    x = Dropout(0.5)(x)
-    x = Dense(4096, activation='relu')(x)
-    # Softmax layer Not Necessary
-    return Model(input, x)
+    base_model = VGG16(input_shape=input_shape)
+    base_model.layers.pop()
```

Comment:
It looks a bit of a hack, but I guess it's fine.

Reply:
We can do 
> return Model(input=base_model.input, output=base_model.get_layer('fc2').output)

or 
> return Model(input=base_model.input, output=base_model.get_layer('predictions').input)

if that looks cleaner :smile:

Reply:
It's more or less the same, let's use the current code you wrote.

Reply:
@marco-c I was trying to run the above model but it was giving some error while trying to save the model. I had tested it with the model summary and until first epoch starts. But since model was saved in the end of first epoch I never noticed it (it was unexpected).

Solution is the way I proposed in above comment ie, instead of using
> base_model.layers.pop() 

we use
>Model(input=base_model.input, output=base_model.get_layer('fc2').output)